tests/unit/EventCard.spec.js

```diff
@@ -49,5 +35,41 @@ describe( "EventCard", () => {
 
   it('should render the event card div', () => {
     expect(wrapper.classes()).toContain('event-card');
+  });
+
+  it('should render the heart button and heart image', () => {
+    const button = wrapper.find('.heart-btn')
+    const buttonImg = wrapper.find('.heart-image')
+
+    expect(button.exists()).toBe(true)
+    expect(buttonImg.exists()).toBe(true)
```

Comment:
Good catch here! I need to add the image rendering check to the tests for the search bar.